spec/request/disability_claims_spec.rb

```diff
@@ -48,5 +48,19 @@
       get '/v0/disability_claims/2', nil, 'Authorization' => "Token token=#{session.token}"
       expect(response).to have_http_status(:internal_server_error)
     end
+
+    it 'has waiver_submitted set after requesting a decision' do
+      VCR.use_cassette('evss/claims/claim') do
+        get '/v0/disability_claims/189625', nil, 'Authorization' => "Token token=#{session.token}"
+        expect(JSON.parse(response.body).dig('data', 'attributes', 'waiver_submitted')).to be(false)
+      end
+
+      post '/v0/disability_claims/189625/request_decision', nil, 'Authorization' => "Token token=#{session.token}"
+
+      VCR.use_cassette('evss/claims/claim') do
+        get '/v0/disability_claims/189625', nil, 'Authorization' => "Token token=#{session.token}"
+        expect(JSON.parse(response.body).dig('data', 'attributes', 'waiver_submitted')).to be(true)
+      end
```

Comment:
I like the spirit of this spec, but I think it could be simplified to just the request, then reload the claim and check the  DB field changed, eg.
```ruby
# ensure claim.requested_decision is false before this request in the let block
post '/v0/disability_claims/189625/request_decision', nil, 'Authorization' => "Token token=#{session.token}"
expect(claim.reload.requested_decision).to eq(true)
```
The `waiver_submitted` field logic should be tested separately in the serializer